_data/simple-icons.json

```diff
@@ -6724,6 +6724,11 @@
                 ]
             }
         },
+        {
+            "title": "GSMA",
+            "hex": "000000",
```

Comment:
The company tends to use a red as their brand colour (see all the app store icons!). We might be best placed to use one of those instead of black.